cfdm/field.py

```diff
@@ -92,8 +92,8 @@ def __new__(cls, *args, **kwargs):
         """Store component classes.
 
         .. note:: If a child class requires a different component
-        classes           than the ones defined here, then they must be
-        redefined           in the child class.
+        classes than the ones defined here, then they must           be
```

Comment:
See my first comment with regards to whether such comments are worthwhile, and if so:

```suggestion
        classes than the ones defined here, then they must be
```

and there's an 'a' we should drop in the previous line for the 'classes' plural case.

Reply:
Dropping this text, as per previous comment.